### DISCUSSION TITLE
Destination

### DISCUSSION THREAD

**Discussion by Lieve2810** [osmandapp/OsmAnd#18696] [General]
Hello, on my new smartphone, i can not make a route, because osmand said destination is not on? I try many times, i reset my profile, i reinstall osmand but its still not working. What else can i do please. My phone is Android mi14 rednote

**Comment by mikehgentry** [comment]
If you tap the 'My position' button in the bottom right (it has a kind of crosshair on it), does it centre on you? If that doesn't work, maybe you have to enable location services in Android settings.

If that's working, if you long press on a point on the map, and choose 'Navigation', does it build a route?

**Comment by Lieve2810** [comment]
Good evening,

My problem is that i can make myself a route , point by point. The reply
you send is to go to a location and osmand is making the choice. I want to
make a route on the roads that i want. Before i was tapping along the road.
Can you help me please.


Greets

Lieve

Op za 9 dec. 2023 15:39 schreef mikehgentry ***@***.***>:

**Reply by DmitryAlexei** [reply]
Probably this article may help https://osmand.net/docs/user/plan-route/create-route
